docs/concepts/what-is-an-activity-task-execution.md

```diff
@@ -7,6 +7,10 @@ tags:
   - explanation
 ---
 
-An Activity Task Execution is the execution of an Activity function.
+An Activity Task Execution results from the invocation of Activity Function Definition.
```

Comment:
I wouldn't say it "results from".
An Activity Task Execution is when the Worker uses the context provided from the Activity Task and executes the Activity Function Definition.

Another way to look at it - is it is the period between the ActivityTaskStarted and ActivityTaskCompleted (or other Closed) Events.